src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-java.mdx

```diff
@@ -865,7 +865,10 @@ Once you've reached the `getting-started-java` entity in New Relic:
       title="Screenshot showing the main distributed tracing page"
       src={opentelemetryMainDistributedTracingPage}
     />
-2. From there, find a trace with an error and click to open it:
+2. From there, find a trace with an error and click to open it. 
+    <Callout variant="tip">
+      With the OpenTelemetry Java agent, you might see different result than you'd see with the New Relic APM Java agent. This is because there's a difference in error handling between the two agent types. For more information, review [Handling error rate in OpenTelemetry and New Relic](https://newrelic.com/blog/how-to-relic/error-rate-opentelemetry-and-new-relic).
```

Comment:
@gkhalasi The link to this blog post is broken. Are you working on a team that's planning on publishing that potential page?

Reply:
Ah, disregard this ^ I had local issues accessing this domain. Can you verify what team you're on and I'll add you to the New Relic repo? You can Slack me at @ally sassman.

Reply:
@ally-sassman, I am a New Relic user; who had followed this [documentation](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-java/#java-agent-tutorial)  and found out missing details for error handling differences between OpenTelemetry agent and New Relic agent.